app/scripts/HiGlassComponent.js

```diff
@@ -3346,6 +3348,13 @@ class HiGlassComponent extends React.Component {
     select('body').selectAll('.track-mouseover-menu').remove();
   }
 
+  /**
+   * Handle internally broadcasted click events
+   */
+  appClickHandler(data) {
+    apiPublish('click', data);
```

Comment:
I am re-publishing the click on the API's pubSub. (Notice the naming difference `publish` vs `apiPublish`. In this case it might seem overkill but in general the app could do something in response to the click event before sending it out _into the world_ (i.e., the public API)